pkgs/applications/misc/metar/default.nix

```diff
@@ -2,10 +2,11 @@
 
 stdenv.mkDerivation {
   pname = "metar";
-  version = "20161013.1";
+  version = "unstable-2017-02-17";
 
-  src = fetchgit {
-    url = "https://github.com/keesL/metar.git";
+  src = fetchFromGitHub {
```

Comment:
fetchgit above needs to be changed to fetchFromGitHub

Reply:
Why does neither the LSP nor ``nix-env -qaP`` catch those errors?

Reply:
I don't think rnix-lsp has any special support for nixpkgs yet
I don't know about `nix-env -qaP`, but it's probably a good idea to do a `nixpkgs-review rev HEAD`

Reply:
Yeah, I should have done that.